local-transcriber/README.md

```
@@ -78,17 +87,25 @@
-このリポジトリには、Node.js 版とは別に、Python を使用したローカルでの文字起こしと話者分離を行うスクリプトも含まれています。
+ローカル環境または OpenAI API を利用して音声ファイルの文字起こしを行い、さらにローカルで話者分離を行う高機能な CLI ツールです。
 
 ### 機能
 
-*   OpenAI Whisper (large モデル) を使用した高精度な文字起こし
-*   Resemblyzer と DBSCAN を使用した話者分離
-*   MP3 ファイルを自動的に WAV に変換
-*   出力形式: `@Speaker_N [MM:SS]
-文字起こしテキスト`
+*   **文字起こしエンジン選択:**
+    *   **ローカルモード (デフォルト):** ローカルにインストールされた OpenAI Whisper (large モデル) を使用。
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)